### PR TITLE
Update EIB component doc

### PR DESCRIPTION
**Pull request by dbw7**

### CODE REVIEW

**Review by atanasdinov — APPROVED** [review]

File: asciidoc/components/edge-image-builder.adoc [hunk 1]
```diff
@@ -9,4 +9,42 @@ ifdef::env-github[]
...
+https://github.com/suse-edge/edge-image-builder[Official Repository]
+
+Edge Image Builder (EIB) is a tool that streamlines the process of generating pre-configured, ready-to-deploy (RTD) disk images for bootstrapping machines, allowing for the end to end deployment of the entire SUSE software stack through a single image.
+
+Additionally, while EIB can create RTD images for all provisioning scenarios, EIB demonstrates tremendous value in air-gapped deployments with limited, or completely isolated networks.
+
+
+== How does SUSE Edge use Edge Image Builder?
+
+SUSE Edge uses EIB for the simplified and quick configuration of customized SLEMicro images for a variety of scenarios. These scenarios include the bootstrapping of virtual and bare metal machines with:
```
Comment (agracey) [inline asciidoc/components/edge-image-builder.adoc:21]: SLE Micro (with a space)

File: asciidoc/components/edge-image-builder.adoc [hunk 2]
```diff
@@ -9,4 +9,42 @@ ifdef::env-github[]
...
+
+* Fully air-gapped deployments of K3s/RKE2 Kubernetes (single & multi-node)
+* Fully air-gapped Helm chart and Kubernetes manifest deployments
+* Registration to Rancher via Elemental API
+* Metal3
+* Customized Networking (e.g. static IP, hostname, VLAN's, bonding, etc.)
+* Customized operating system configurations (e.g. users, groups, passwords, ssh-keys, proxies, NTP, custom SSL certificates, etc.)
+* Air-gapped installation of host-level packages (including dependency resolution)
+* SCC Registration
```
Comment (agracey) [inline asciidoc/components/edge-image-builder.adoc:30]: "Registration with SUSE Customer Center (SCC)" ?

File: asciidoc/components/edge-image-builder.adoc [hunk 3]
```diff
@@ -9,4 +9,42 @@ ifdef::env-github[]
...
+
+
+== How does SUSE Edge use Edge Image Builder?
+
+SUSE Edge uses EIB for the simplified and quick configuration of customized SLEMicro images for a variety of scenarios. These scenarios include the bootstrapping of virtual and bare metal machines with:
+
+* Fully air-gapped deployments of K3s/RKE2 Kubernetes (single & multi-node)
+* Fully air-gapped Helm chart and Kubernetes manifest deployments
+* Registration to Rancher via Elemental API
+* Metal3
```
Comment (agracey) [inline asciidoc/components/edge-image-builder.adoc:26]: `Metal^3^` (I only just learned this...)

**Review by agracey — APPROVED** [review]
Thank you! Only a few comments on branding stuff 😆

**Pull request merged**